File: Test_Git/scala-maven-deom/src/main/com.huntkey.scala/vo/UserInfoDto.scala
package vo

/**
  * Created by liuwens on 2017/9/13.
  */
class UserInfoDto
{

  var userName:String = "";

  private var userAge:Int = 0;

  //如果定义的方式没有(), 则引用时也不能有
  def getUserAge:Int = this.userAge;

  def setUserAge(age:Int):Unit =
  {
    this.userAge = age;
  }


}

File: Test_Git/eureka-feign-demo/src/main/java/com/huntkey/DTO/UserInfoDTO.java
package com.huntkey.DTO;


import com.alibaba.fastjson.annotation.JSONField;

/**
 * Created by liuwens on 2017/8/18.
 */
public class UserInfoDTO
{
    @JSONField(name = "user_name")
    private String userName;

    @JSONField(name = "user_age")
    private int userAge;

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName=userName;
    }

    public int getUserAge() {
        return userAge;
    }

    public void setUserAge(int userAge) {
        this.userAge=userAge;
    }
}
